script: add script to setup demo instance

edivalentinitu opened the pull request:
- add a script that sets the whole demo instance up
- also optional steps available to add to this script (e.g publications, demo_users)

edivalentinitu committed:
script: add script to setup demo instance

mb-wali reviewed (changes requested):
the script is nice, But the issue i see is updating the commands from time to time, its complex for some devs.
i would suggest something like [This script](https://github.com/inveniosoftware/demo-inveniordm/blob/master/demo-inveniordm/wipe_recreate.sh)
and reference a command in the README after running the docker-compose 
e.g
```docker exec repository_web-api_1 ./wipe_recreate.sh```

edivalentinitu commented:
> the script is nice, But the issue i see is updating the commands from time to time, its complex for some devs. i would suggest something like [This script](https://github.com/inveniosoftware/demo-inveniordm/blob/master/demo-inveniordm/wipe_recreate.sh) and reference a command in the README after running the docker-compose e.g `docker exec repository_web-api_1 ./wipe_recreate.sh`

Added now the possibility to run wipe_recreate.sh after manually running docker compose. Also to setup publications. These were the 2 sets of _many_ commands. These 2 new scripts are also used in the start-app script. Also updated the readme.

mb-wali commented on README.md:
```diff
@@ -8,7 +8,21 @@ Nginx example configuration can be found under [nginx](nginx).
+
+### Initialization
+
+### 1. Fast way of starting demo instance:
```
please remove this from README. you can keep the script. but remove it from README - lets have just the **step by step**

mb-wali commented on README.md:
```diff
@@ -19,11 +33,19 @@ Once running, visit https://127.0.0.1 in your browser.
+**Note**: This might not work just by running docker-compose. You might need to run the following steps in order for the application to show up in the browser.
 
 To set up your database, indexes, and related components, follow these steps.  
 **Note:** These commands should only be run during the initial setup. Running them multiple times may result in data loss.
```
```suggestion
**Warning:** These commands should only be run during the initial setup. Running them multiple times may result in data loss.
```

mb-wali reviewed (approved):
left some comments, can b merge after
